server/parameters/config.rs

```diff
@@ -80,15 +80,19 @@ impl Config {
             storage: StorageConfig { data: data_directory.to_owned() },
         }
     }
+
+    pub fn get_server_config(&self) -> &ServerConfig {
+        &self.server
+    }
```

Comment:
We should unlearn the `get_` prefixes now
```suggestion
    pub fn server_config(&self) -> &ServerConfig {
        &self.server
    }
```

Reply:
For some reason, I thought that the usage of `get_` is actually the preferred naming! Thanks